deno_webgpu/compute_pass.rs

```diff
@@ -32,7 +32,7 @@ pub fn op_webgpu_compute_pass_set_pipeline(
         .get::<WebGpuComputePass>(compute_pass_rid)?;
 
     state
-        .borrow::<wgpu_core::global::Global>()
+        .borrow::<super::Instance>()
```

Comment:
`super::Instance` wraps a `wgpu_core::global::Global` and is also actually present in the runtime state. it's used in locations that continued to pass tests after 77a83fb0dd9f2295f25e99a850b9a031738925c3 and using it consistently seems to correct the test panics.

Reply:
several op functions in command_encoder, render_pass, and compute_pass use some `gfx_ok` and `gfx_put` macros.

i didn't add or remove any calls, but i noticed use seems to have become more inconsistent over time.